apps/library/static/css/autocomplete.css

```
@@ -2,3 +2,8 @@
 .select2-results__options {
    max-height: 450px !important;
 }
+/* resize button for preview page */
+a.button {
+   padding: 10px 14px;
+   background: #00CC99;
```

Давай чуть цвет поменяем
```suggestion
   background: #759e4a;
```
и добавим при наведении изменение цвета
```
a.button:hover {
    background: #435b2a;
}
```